controller/src/api/serializers/customer_serializers.py

```diff
@@ -167,3 +171,26 @@ class SectorGetSerializer(serializers.Serializer):
     name = serializers.CharField(max_length=250)
     industries = SectorIndustry(many=True)
 
+
+class CustomerQurySerializer(serializers.Serializer):
```

Comment:
Typo - Should be CustomerQuerySerializer

Reply:
ah, good catch, updated now